apps/api/package.json

```diff
@@ -29,9 +29,14 @@
     "database:reset": "ts-node src/database/database-reset.ts"
   },
```

Comment:
Remove all files from PR which are not related to your task.

Reply:
okay, i have merged the current main branch in my local test branch